### PR TITLE
staging the removal of eventingv1beta1

### PR DESCRIPTION
**Pull request by vaikas**
Fixes #

## Proposed Changes

- Staging changes after: https://github.com/knative/eventing/pull/5201
-
-

**Release Note**

```release-note
🗑️ Remove {eventing,flows,messaging} v1beta1 APIs
```

**Docs**

### CODE REVIEW

**Comment by matzew** [comment]
@vaikas thanks a lot for this PR!

File: test/conformance/channel_tracing_test.go [hunk 1]
```diff
@@ -25,17 +25,19 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"knative.dev/eventing/test/conformance/helpers"
 	testlib "knative.dev/eventing/test/lib"
-	"knative.dev/eventing/test/lib/resources"
 
 	contribtest "knative.dev/eventing-kafka/test"
 )
 
+// Eventing channels are v1, but Kafka channel is v1beta1.
+const kafkaChannelAPIVersion = "messaging.knative.dev/v1beta1"
```
Comment (matzew) [inline test/conformance/channel_tracing_test.go:33]: Can we make use of this, for exactly the reasons you added a comment, here:

https://github.com/knative-sandbox/eventing-kafka/blob/d8d8d2333cd7735c7dd538825ea6a5ed4b2bf9f3/test/upgrade/test_main.go#L34

:pray:

Comment (matzew) [reply test/conformance/channel_tracing_test.go:33]: related: 

* https://github.com/knative-sandbox/eventing-kafka/pull/467/files#r605554669
* https://github.com/knative/eventing/pull/5201/#pullrequestreview-629025730

**Comment by vaikas** [comment]
interesting failure:
```
=== FAIL: test/e2e TestKafkaSourceReconciler (17.53s)
    test_runner.go:153: namespace is : "test-kafka-source-reconciler-5hbxq"
    creation.go:77: Creating channel &TypeMeta{Kind:InMemoryChannel,APIVersion:messaging.knative.dev/v1,}-e2e-rt-channel
    tracker.go:153: Waiting for all KResources to become ready
    deletion.go:31: Failed to delete the resource "e2e-rt-channel" : conversion webhook for messaging.knative.dev/v1, Kind=InMemoryChannel failed: Post https://eventing-webhook.knative-eventing.svc:443/?timeout=30s: x509: certificate signed by unknown authority
    k8s_events.go:66: EventListener stopped, 9 events seen

```

/test pull-knative-sandbox-eventing-kafka-integration-test-channel-consolidated

**Comment by vaikas** [comment]
/test pull-knative-sandbox-eventing-kafka-integration-test-channel-consolidated
retrying, I don't understand this failure

**Comment by vaikas** [comment]
Again the cert issue, or I'm reading the logs wrong.
/test pull-knative-sandbox-eventing-kafka-integration-test-channel-consolidated

**Comment by matzew** [comment]
/hold

until we have cut 0.22

**Comment by matzew** [comment]
/close

was replaced by #513

**Pull request closed without merging**